### PR TITLE
Pot/Potluck status report

### PR DESCRIPTION
**Pull request by hny-gd**

### CODE REVIEW

**Review by lsalvadore — APPROVED** [review]
Approved.

@sergio-carlavilla @dbaio: do you mentor approve the commit? Thanks

File: website/content/en/status/report-2023-04-2023-06/pot.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Links: +
+link:https://github.com/bsdpot[Pot organization on GitHub] URL: link:https://github.com/bsdpot[]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Bretton Vine (Potluck) <bv@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During the last quarter, link:https://github.com/bsdpot/pot/releases/tag/0.15.5[pot 0.15.5] was released, containing a number of bugfixes and attribute features from various contributors. It will be available on the 2023Q3 quarterly package set.
```
Comment (pauamma) [inline website/content/en/status/report-2023-04-2023-06/pot.adoc:12]: Can you clarify what you mean by "attribute features"?

File: website/content/en/status/report-2023-04-2023-06/pot.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases Nomad.
+
+All Potluck containers have been rebuilt as FreeBSD 13.2 based images and are signed with link:https://github.com/bsdpot/pot/pull/242[pot signify] now.
+
+link:https://honeyguide.eu/posts/ansible-pot-foundation/[A Beginner's Guide to Building a Virtual Datacenter on FreeBSD with Ansible, Pot and More] has been written, explaining how a complex environment based on Pot and Potluck can be deployed with Ansible playbooks, including example nodes like MariaDB, Prometheus, Grafana, Nginx, OpenLDAP or Traefik and container orchestration managed by Nomad and Consul.
+
+A link:https://github.com/hashicorp/nomad/pull/13343[patch by the pot team] to improve nomad security, a scheduler and orchestrator which supports pot through
+link:https://cgit.freebsd.org/ports/tree/sysutils/nomad[sysutils/nomad-pot-driver], has been accepted upstream and will be part of nomad 1.6.0.
+
+
```
Comment (pauamma) [inline website/content/en/status/report-2023-04-2023-06/pot.adoc:23]: Spurious linebreak.

File: website/content/en/status/report-2023-04-2023-06/pot.adoc (hunk 1 above)
Comment (hny-gd) [reply website/content/en/status/report-2023-04-2023-06/pot.adoc]: Added explanation and link.

File: website/content/en/status/report-2023-04-2023-06/pot.adoc (hunk 2 above)
Comment (hny-gd) [reply website/content/en/status/report-2023-04-2023-06/pot.adoc:21]: Removed.

**Comment by hny-gd** [comment]
Thanks, @pauamma, made two changes based on your feedback.

**Review by pauamma — APPROVED** [review]

File: website/content/en/status/report-2023-04-2023-06/pot.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Bretton Vine (Potluck) <bv@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During the last quarter, link:https://github.com/bsdpot/pot/releases/tag/0.15.5[pot 0.15.5] was released, containing a number of bugfixes and link:https://github.com/bsdpot/pot/pull/263[features to set attributes (i.e. jail sysctl variables)] from various contributors. It will be available on the 2023Q3 quarterly package set. 
+
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases Nomad.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-04-2023-06/pot.adoc:14]:
```suggestion
Potluck aims to be to FreeBSD and Pot what Dockerhub is to Linux and Docker: a repository of Pot flavours and complete container images for usage with Pot and in many cases Nomad.
```

File: website/content/en/status/report-2023-04-2023-06/pot.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+All Potluck containers have been rebuilt as FreeBSD 13.2 based images and are signed with link:https://github.com/bsdpot/pot/pull/242[pot signify] now.
+
+link:https://honeyguide.eu/posts/ansible-pot-foundation/[A Beginner's Guide to Building a Virtual Datacenter on FreeBSD with Ansible, Pot and More] has been written, explaining how a complex environment based on Pot and Potluck can be deployed with Ansible playbooks, including example nodes like MariaDB, Prometheus, Grafana, Nginx, OpenLDAP or Traefik and container orchestration managed by Nomad and Consul.
+
+A link:https://github.com/hashicorp/nomad/pull/13343[patch by the pot team] to improve nomad security, a scheduler and orchestrator which supports pot through link:https://cgit.freebsd.org/ports/tree/sysutils/nomad[sysutils/nomad-pot-driver], has been accepted upstream and will be part of nomad 1.6.0.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-04-2023-06/pot.adoc:20]:
```suggestion
A link:https://github.com/hashicorp/nomad/pull/13343[patch by the pot team] to improve Nomad security, a scheduler and orchestrator which supports Pot through link:https://cgit.freebsd.org/ports/tree/sysutils/nomad[sysutils/nomad-pot-driver], has been accepted upstream and will be part of Nomad 1.6.0.
```

File: website/content/en/status/report-2023-04-2023-06/pot.adoc [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+During the last quarter, link:https://github.com/bsdpot/pot/releases/tag/0.15.5[pot 0.15.5] was released, containing a number of bugfixes and link:https://github.com/bsdpot/pot/pull/263[features to set attributes (i.e. jail sysctl variables)] from various contributors. It will be available on the 2023Q3 quarterly package set. 
+
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases Nomad.
+
+All Potluck containers have been rebuilt as FreeBSD 13.2 based images and are signed with link:https://github.com/bsdpot/pot/pull/242[pot signify] now.
+
+link:https://honeyguide.eu/posts/ansible-pot-foundation/[A Beginner's Guide to Building a Virtual Datacenter on FreeBSD with Ansible, Pot and More] has been written, explaining how a complex environment based on Pot and Potluck can be deployed with Ansible playbooks, including example nodes like MariaDB, Prometheus, Grafana, Nginx, OpenLDAP or Traefik and container orchestration managed by Nomad and Consul.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-04-2023-06/pot.adoc:18]:
```suggestion
link:https://honeyguide.eu/posts/ansible-pot-foundation/[A Beginner's Guide to Building a Virtual Datacenter on FreeBSD with Ansible, Pot and More] has been written, explaining how a complex environment based on Pot and Potluck can be deployed with Ansible playbooks, including example nodes like MariaDB, Prometheus, Grafana, nginx, OpenLDAP or Traefik and container orchestration managed by Nomad and Consul.
```

File: website/content/en/status/report-2023-04-2023-06/pot.adoc [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+Contact: Bretton Vine (Potluck) <bv@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During the last quarter, link:https://github.com/bsdpot/pot/releases/tag/0.15.5[pot 0.15.5] was released, containing a number of bugfixes and link:https://github.com/bsdpot/pot/pull/263[features to set attributes (i.e. jail sysctl variables)] from various contributors. It will be available on the 2023Q3 quarterly package set. 
+
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases Nomad.
+
+All Potluck containers have been rebuilt as FreeBSD 13.2 based images and are signed with link:https://github.com/bsdpot/pot/pull/242[pot signify] now.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-04-2023-06/pot.adoc:16]:
```suggestion
All Potluck containers have been rebuilt as FreeBSD 13.2 based images and are signed with link:https://github.com/bsdpot/pot/pull/242[Pot signify] now.
```

**Review by grahamperrin — CHANGES_REQUESTED** [review]

**Comment by hny-gd** [comment]
Thanks a lot, @grahamperrin for your prettifying, I have committed all your suggested changes.

**Comment by lsalvadore** [comment]
Committed with commit 91906cbf.

**Pull request closed without merging**